.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.production.md

```diff
@@ -0,0 +1,30 @@
+## Release to Production
+
+This Pull Request affects a release from val to production.
+
+#### Closed Issues in this release
```

Comment:
@ayoneeyee I'd like to remove this "Closed Issues" section.  
The idea behind this section was to have th creator list out each PR/issue that's being merged.  But, we've found that our PR hygiene solves this for us.  Git shows all commits included in the PR, and since we always squash and merge to master, the commit message is auto-generated with a PR link.  So, Github actually generates a 100% accurate list of all commits going in, so this section isn't needed (and is human error prone).  Same thing for the val template... i'll make a note there too for completeness.

So yeah this looks good, but I vote you rip out this section